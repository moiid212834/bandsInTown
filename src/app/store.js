import { configureStore } from '@reduxjs/toolkit';
import themeReducer from '../features/theme/ThemeSlice';
import artistReducer from '../features/artists/ArtistSlice';
import eventsReducer from '../features/events/EventsSlice';

/**
 * Configuring the store using redux toolkit and passing reducers
 */
export const store = configureStore({
  reducer: {
    theme: themeReducer,
    artists: artistReducer,
    events: eventsReducer,
  },
});
