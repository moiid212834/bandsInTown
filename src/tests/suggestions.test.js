import React from 'react';
import {Provider} from 'react-redux';
import {store} from '../app/store';
import Suggestions from '../components/artistSuggestions';
import {BrowserRouter as Router} from 'react-router-dom';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
    const tree = renderer
        .create(
        <Provider store={store}>
            <Router>
                <Suggestions></Suggestions>
            </Router>
        </Provider>
    )
        .toJSON();
    expect(tree).toMatchSnapshot();
});
