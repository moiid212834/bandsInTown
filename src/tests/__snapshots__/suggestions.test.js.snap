// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="MuiBox-root css-1r1aq4"
>
  <h4
    className="MuiTypography-root MuiTypography-h4 css-pxw7rc-MuiTypography-root"
  >
    Suggested Artists
  </h4>
  <div
    style={
      Object {
        "display": "none",
      }
    }
  >
    <div
      className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 MuiGrid-spacing-md-3 css-1xgjpwf-MuiGrid-root"
    >
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2 MuiGrid-grid-sm-3 MuiGrid-grid-md-3 css-1plh1vb-MuiGrid-root"
      >
        <div
          style={
            Object {
              "textAlign": "center",
            }
          }
        >
          <span
            className="MuiSkeleton-root MuiSkeleton-circular MuiSkeleton-pulse css-fomgco-MuiSkeleton-root"
            style={
              Object {
                "height": "150px",
                "margin": "20px auto",
                "width": "150px",
              }
            }
          />
          <span
            className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse css-1n7gpw1-MuiSkeleton-root"
            style={
              Object {
                "height": undefined,
                "width": "100%",
              }
            }
          />
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2 MuiGrid-grid-sm-3 MuiGrid-grid-md-3 css-1plh1vb-MuiGrid-root"
      >
        <div
          style={
            Object {
              "textAlign": "center",
            }
          }
        >
          <span
            className="MuiSkeleton-root MuiSkeleton-circular MuiSkeleton-pulse css-fomgco-MuiSkeleton-root"
            style={
              Object {
                "height": "150px",
                "margin": "20px auto",
                "width": "150px",
              }
            }
          />
          <span
            className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse css-1n7gpw1-MuiSkeleton-root"
            style={
              Object {
                "height": undefined,
                "width": "100%",
              }
            }
          />
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2 MuiGrid-grid-sm-3 MuiGrid-grid-md-3 css-1plh1vb-MuiGrid-root"
      >
        <div
          style={
            Object {
              "textAlign": "center",
            }
          }
        >
          <span
            className="MuiSkeleton-root MuiSkeleton-circular MuiSkeleton-pulse css-fomgco-MuiSkeleton-root"
            style={
              Object {
                "height": "150px",
                "margin": "20px auto",
                "width": "150px",
              }
            }
          />
          <span
            className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse css-1n7gpw1-MuiSkeleton-root"
            style={
              Object {
                "height": undefined,
                "width": "100%",
              }
            }
          />
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2 MuiGrid-grid-sm-3 MuiGrid-grid-md-3 css-1plh1vb-MuiGrid-root"
      >
        <div
          style={
            Object {
              "textAlign": "center",
            }
          }
        >
          <span
            className="MuiSkeleton-root MuiSkeleton-circular MuiSkeleton-pulse css-fomgco-MuiSkeleton-root"
            style={
              Object {
                "height": "150px",
                "margin": "20px auto",
                "width": "150px",
              }
            }
          />
          <span
            className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse css-1n7gpw1-MuiSkeleton-root"
            style={
              Object {
                "height": undefined,
                "width": "100%",
              }
            }
          />
        </div>
      </div>
    </div>
  </div>
  <div
    className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 MuiGrid-spacing-md-3 css-1xgjpwf-MuiGrid-root"
  />
</div>
`;
